src/graph/sampler.cc

```diff
@@ -532,14 +538,16 @@ DGL_REGISTER_GLOBAL("nodeflow._CAPI_NodeFlowFree")
     delete nflow;
   });
```

Comment:
This _CAPI_NodeFlowFree is actually not needed anymore. If I haven't deleted it before, you could delete it here.